### DISCUSSION TITLE
Random radius of Meshinstance Spheres

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#356] [Answers & Questions] [answered]
I am unable to achieve random radius size of spheres connected to a **MeshInstancer** taking points from a **PointsSphereRandom** Op. Some of the spheres get flat like chips instead of remaining round:

![Untitled](https://user-images.githubusercontent.com/94358787/192157827-03b29c70-c4ad-4733-8500-073c0f01e358.jpg)

**Comment by monolithMktg** [comment, marked as answer]
Ok I was able to solve it like this. But is it the ideal way to achieve it? As the **Scale Array** length = **Points Array** length, not number of points.
So the number of points are 100, and the array length is 300, which gets trickled own all the way to plug into Scale Array value of **Mesh Instancer** Op with the same 300 length array from **ArrayPack3** Op.

![Untitled](https://user-images.githubusercontent.com/94358787/192159860-eeb701f2-fb32-4d82-9099-90f4aaf16f23.jpg)

**Comment by pandrr** [comment]
yes this is fine, why not ?

**Reply by monolithMktg** [reply]
Just trying to figure if there was a better way. Now that you have confirmed, this is it :)
